package.json

```diff
@@ -9,7 +9,7 @@
     "test:debug": "node --inspect node_modules/.bin/jest --runInBand --silent",
     "coverage": "jest --coverage --silent",
     "dev": "nodemon --max-old-space-size=8192 src/app.js",
-    "start": "node --max-old-space-size=8192 src/app.js",
+    "start": "node --unhandled-rejections=strict --max-old-space-size=8192 src/app.js",
```

Comment:
It transforms unhandled rejections from just showing up in the log to actually throwing the error that caused the rejection, I was debugging some problems with the validation yaml that weren't really apparent (the api still seemed to work) until I noticed some lines about unhandled rejections.

I thought it could be useful to keep this after I was done with debugging that.

Reply:
Does "actually throwing the error" means causing the API process to stop?